inst/EpiModel2.rmd

```diff
@@ -0,0 +1,684 @@
+formation <- ~edges + nodematch("group")
+target.stats <- c(30, 0)
+```
+
+## Epidemic Type
+
+When `control.net` input parameter `type` is `NULL`,
+
+```{r echo = TRUE, eval = FALSE}
+control <- control.net(type = NULL, nsims = 10, nsteps = 10)
+```
+
+the user must specify all base modules, updating code as follows:
+
+```{r echo = TRUE, eval = FALSE}
+control <- control.net(type = NULL, nsims = 10, nsteps = 10,
+                       resim_net.FUN = resim_nets, infection.FUN = infection.net,
+                       recovery.FUN = recovery.net, prevalence.FUN = prevalence.net,
+                       departures.FUN = departures.net, arrivals.FUN = arrivals.net)
+```
+
+## Depend to resimulate.network
+
+The `control.net` parameter used to resimulate the network at each time has been renamed from depend
+
+```{r echo = TRUE, eval = FALSE}
+control <- control.net(type = "SI", nsims = 10, nsteps = 10, depend = TRUE)
+```
+
+to resimulate.network
+
+```{r echo = TRUE, eval = FALSE}
+control <- control.net(type = "SI", nsims = 10, nsteps = 10, resimulate.network = TRUE)
+```
+
+## Mode to Group Parameters
+
+Mode syntax has been updated to group syntax where appropriate. The following:
+
+```{r echo = TRUE, eval = FALSE}
+init <- init.net(i.num = 50, i.num.m2 = 50)
+param <- param.net(inf.prob = 0.1, inf.prob.m2 = 0.1,
+                   rec.rate = 0.02, rec.rate.m2 = 0.02,
+                   act.rate = 5)
+```
+
+should be updated to
+
+```{r echo = TRUE, eval = FALSE}
+init <- init.net(i.num = 50, i.num.g2 = 50)
+param <- param.net(inf.prob = 0.1, inf.prob.g2 = 0.1,
+                   rec.rate = 0.02, rec.rate.g2 = 0.02,
+                   act.rate = 5)
+```
+
+## Using accessor functions
+
+The new asseccor functions are easy to use utilities for getting, setting and appending
+data, replacing `$`  and `[[ ]]` operations.
+
+```{r echo = TRUE, eval = FALSE}
+dat <- list()
+partners <- rpois(100, 1)
+dat$attr$partners <- partners
+```
+
+or 
+
+```{r echo = TRUE, eval = FALSE}
+dat <- list()
+partners <- rpois(100, 1)
+dat[["attr"]][["partners"]] <- partners
+```
+
+should be updated in script to:
+
+```{r echo = TRUE, eval = FALSE}
+dat <- lisit()
+partners <- rpois(100, 1)
+dat <- set_attr(dat, "partners", partners)
+```
+
+Similarly for retrieval and setting of list elements.
+
+```{r echo = TRUE, eval = FALSE}
+partners <- dat$attr$partners
+```
+
+or 
+
+```{r echo = TRUE, eval = FALSE}
+partners <- dat[["attr"]][["partners"]]
+```
+
+should be updated to
+
+```{r echo = TRUE, eval = FALSE}
+partners <- get_attr(dat, "partners")
+```
+
+And
+
+```{r echo = TRUE, eval = FALSE}
+newPartners <- rpois(2, 1)
+dat$attr$partners[101:102] <- newPartners
+```
+
+or 
+
+```{r echo = TRUE, eval = FALSE}
+newPartners <- rpois(2, 1)
+dat[["attr"]][["partners"]][101:102] <- newPartners
+```
+
+should be updated to
+
+```{r echo = TRUE, eval = FALSE}
+newPartners <- rpois(2, 1)
+dat <- append_attr(dat, "partners", newPartners, 2)
+```
+
```

Comment:
Similar comment for these

Reply:
Not sure what this comment is referencing.